### ISSUE TITLE
Hidden controls SC3.2.7 - case where the hover is over a related component

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#1241]
The Understanding document for this SC (2nd paragraph under "Intent") says:
> may not be able to find controls needed to progress if they are hidden until focus is placed on them or a pointer hovers over them.

In real life you don't necessarily hover over the controls themselves, but over a nearby component. That case is not caught by the above words in the Understanding - although it should be covered. (After all, how can you hover over something you can't see, though some websites manage to do it nevertheless!)

Take the inbox in Gmail. Hovering over one of the email descriptions in the list reveals 4 controls that then overlay the last inch or so of the text. (You can see this is actually what was in mind in the third example in the Examples section). 

The point here is that the pointer doesn't hover over the controls - they can't be seen at that point. The user hovers over the text of the description, which is visible. (Gmail isn't the only website to do this.)

So the wording of the Understanding doc needs to include this pattern as well. I suggest changing the lasts few words of the above extract to something like the following to catch this situation:
> until focus is placed on, or a pointer hovers over, the controls or other related component.

**Comment by alastc** [comment]
Hi Guy,

This success criteria has been heavily overhauled, and I don't think this issue applies any more.

The [latest version](https://raw.githack.com/w3c/wcag/wcag22-visible-controls-update/understanding/22/visible-controls.html) uses a different approach which focuses on the information for that control being available is visible.

Please do raise any new issues on the new text, it will be going out for review again.
